pkg/agent/utils.go

```diff
@@ -576,6 +576,21 @@ func strKeyValToMapBool(str string, strDelim string, pairDelim string) map[strin
 	return m
 }
 
+func remoteFeatureGateString(featureGates string, key string) string {
```

Comment:
is this supposed to be "remove" instead of "remote"?

Reply:
oops :)